### DISCUSSION TITLE
Cannot read properties of undefined

### DISCUSSION THREAD

**Discussion by jigujigu** [faker-js/faker#1877] [Q&A] [answered]
Hi I got error  :
ERROR: Cannot read properties of undefined (reading 'email')

```
const faker = require('@faker-js/faker/locale/en');

const generateUser = () => ({
  email: faker.internet.email(),
  password: faker.internet.password(),
  active: faker.datatype.boolean()
});
```
thanks

**Comment by Shinigami92** [comment, marked as answer]
```diff
- const faker = require('@faker-js/faker/locale/en');
+ const { faker } = require('@faker-js/faker/locale/en');
```

**Reply by matthewmayer** [reply]
I wonder if there is any way we could throw a more useful error message in this case. As I'm sure it's a very common error.

**Reply by Shinigami92** [reply]
Feel free to suggest this to NodeJS, but this does not have something to do with FakerJS
because we officially decided to not support default exports to outside by design

**Comment by jigujigu** [comment]
Thanks..
